### PR TITLE
Add case identifier

### PR DESCRIPTION
**Pull request by whuang7000**
# ✨ New in this PR ✨

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [X] Add identifier to firebase for existing case types
 - [X] Atomic write of numcases + case

## How can the reviewer test your code? 👩‍💻
run through intake form questions, check firebase for case with unique identifier

## Any bugs you encountered or still having trouble with? 🐛
_describe how you solved them such that someone can follow your explanation_ 

## Resources 📔
_link online resources and related PRs_ 

🧜‍♀️ cc: @shannonbonet @phoebeli23

### CODE REVIEW

**Review by j-keating0002 — CHANGES_REQUESTED** [review]
@whuang7000 i tried submitting two new daca renewal cases (with the names test and test2) and the ids ended up being undefined-4 and undefined-5 is there an example in the firebase where the id was returned correctly? also for the responses for questions like the type of application (ex. daca renewal, citizenship), let's add an empty default answer like "--" so we remove the possibility of the undefined object error. other than that amazing work!!

File: src/components/Questions/AdditionalQuestionManager/AdditionalQuestionManager.tsx [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
 import { TextSubtitle, TextRegularWhite } from 'assets/fonts/Fonts';
 import { ButtonDarkBlue } from 'assets/Components';
-import {
-  Question,
-  Client,
-  QuestionManagerProps,
-  Case,
-  CaseStatus,
-} from 'types/types';
+import { Question, Client, QuestionManagerProps } from 'types/types';
```
Comment (bry-gavino) [inline src/components/Questions/AdditionalQuestionManager/AdditionalQuestionManager.tsx:11]: Thank you for removing unused imports!

**Comment by bry-gavino** [comment]
Everything looks like it's working on my end, but it seems like there are some undefined issues on Jared's end. I'm going to wait a bit on merging this just to make sure it's nothing inherently wrong with the code.

Other than that, all the code and firebase storage looks great--thank you so much Will!!

**Review by j-keating0002 — APPROVED** [review]
just kidding it was an error on my end.. this looks AMAZING! thanks for taking the time to implement one of the new features we got. ur a g :)

**Pull request merged**